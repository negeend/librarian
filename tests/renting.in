RENT 100000 111111
rent 100000 111111
rEnT 3847y4 skdncs
RELINQUISH 100000 111111
relinquish 100000 111111
RelINqUisH 100000 111111
RELINQUISH ALL 100000
relinquish all 100000
ADD MEMBER Jimmy John
ADD MEMBER Rick Ross
RENT 100000 111111
rent 100000 111111
rEnT wefbhd qiwhqdbn
RELINQUISH 100000 111111
relinquish 100000 111111
RelINqUisH skfnsd skrbfsr
RELINQUISH ALL 1938273
RelINqUisH All jdsbhfs
RELINQUISH ALL 100000
ADD COLLECTION books.csv
RENT 100005 111111
rent 100000 123456
rent 103892 183924
RenT hello hello
RELINQUISH 132844 111111
relinquish 100000 12e324
relinquish 382478 293822
RENT 100000 111111
Rent 100001 111111
rent 100001 111112
RELINQUISH 100000 111111
RELINQUISH 100000 111116
RELINQUISH ALL 100001
EXIT