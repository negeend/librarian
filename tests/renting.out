user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: Success.

user: Success.

user: No books in system.

user: No books in system.

user: No books in system.

user: No books in system.

user: No books in system.

user: No books in system.

user: No such member in system.

user: No such member in system.

user: Success.

user: 17 books successfully added.

user: No such member in system.

user: No such book in system.

user: No such member in system.

user: No such member in system.

user: No such member in system.

user: No such book in system.

user: No such member in system.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Unable to return book.

user: Success.

user: Ending Library process.
